inst/templates/ad_adcm.R

```diff
@@ -16,16 +16,20 @@ library(lubridate)
 data("cm")
 data("adsl")
 data("ex")
-data("adsl")
 
 # ---- Derivations ----
+
+# Derive flags
 adcm <- cm %>%
 
   # Join supplementary qualifier variables
   # derive_vars_suppqual(suppcm)
```

Comment:
Please add a trailing `%>%` such that when this is uncommented it actually works.